doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,18 @@
 
 <h3>New features since last release</h3>
 
+* `qml.QNode` and `qml.qnode` now accept two new keyword  arguments: `postselect_mode` and `mcm_method`.
+  These keyword arguments can be used to configure how the device should behave when running circuits with
+  mid-circuit measurements.
+  [(#5679)](https://github.com/PennyLaneAI/pennylane/pull/5679)
+
+  * `postselect_mode="hw-like"` will indicate to devices to discard invalid shots when postselecting
+    mid-circuit measurements. Use `postselect_mode="fill-shots"` to return invalid shots, which will be replaced
+    by dummy values.
```

Comment:
Wait, I thought `fill-shots` would produce all valid samples?

Quoting from the full documentation:
> If ``postselect_mode="fill-shots"`` is used, then the postselected value will be picked unconditionally, and all samples will be considered valid.

The PR description also contradicts the doc statement, but I think the doc statement is the correct one.

Reply:
Thanks for spotting this! It's an artifact from an older implementation that I must've missed.